packages/loading-spinner/LoadingSpinner/src/LoadingSpinner.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
+import styles from "./styles.scss"
+
+type size = "sm" | "md"
+
+export interface LoadingSpinnerProps {
+  accessibilityLabel: string
+  /**
+   * Generally use "md" unless spinner is inside a form field
+   * @default "md"
+   */
+  size?: size
+  children?: React.ReactNode
```

Comment:
Perhaps to fix later, but it looks like `children` doesn't do anything for the component so we should be able to remove it (but technically it's a breaking change I guess)

Reply:
Yeah I was thinking that was an odd one, was going to confirm with peeps before axing it but will definitely do it in a separate PR :)